doc/radosgw/bucket_logging.rst

```diff
@@ -131,16 +131,16 @@ based on `AWS Logging Record Format`_.
   - bytes sent (or dash if zero)
   - object size (or dash if zero)
   - total time (not supported, always a dash)
-  - turnaround time (not supported, always a dash)
-  - referrer (not supported, always a dash)
-  - user agent (not supported, always a dash)
+  - turnaround time in milliseconds
+  - referer (or dash if empty)
```

Comment:
referrer

Reply:
from wikipedia: "In HTTP, "Referer" (a misspelling of "Referrer") is an optional HTTP header field ..." :-)

Reply:
feh